src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public abstract class Figure extends ColorSupplier implements AreaCalculator, Drawer {
+    private double area;
+    private Color color;
+
```

Comment:
please add a constructor that accepts color

Reply:
add